backend/src/packages/articles/libs/helpers/get-where-genre-id-query.helper.ts

```diff
@@ -0,0 +1,13 @@
+import { type QueryBuilder } from 'objection';
+
+import { type ArticleModel } from '../../article.model.js';
+
+const getWhereGenreIdQuery = (genreId?: number | null) => {
+  return (builder: QueryBuilder<ArticleModel, ArticleModel[]>): void => {
+    if (genreId && genreId > 0) {
```

Comment:
Same about validation on positive number
```suggestion
    if (genreId) {
```